engine/consensus/dkg/reactor_engine.go

```diff
@@ -126,8 +126,8 @@ func (e *ReactorEngine) EpochSetupPhaseStarted(currentEpochCounter uint64, first
 
 // EpochCommittedPhaseStarted handles the EpochCommittedPhaseStarted protocol
 // event by checking the consistency of our locally computed key share.
-func (e *ReactorEngine) EpochCommittedPhaseStarted(currentEpochCounter uint64, _ *flow.Header) {
-	e.handleEpochCommittedPhaseStarted(currentEpochCounter)
+func (e *ReactorEngine) EpochCommittedPhaseStarted(currentEpochCounter uint64, first *flow.Header) {
+	e.handleEpochCommittedPhaseStarted(currentEpochCounter, first)
```

Comment:
Is there a reason for introducing `handleEpochCommittedPhaseStarted` ?

Reply:
Previously it spawned a goroutine, which was the original reason. Even without the goroutine though, I think there's a case for separating the public function implementing the interface from the internal handling logic.